client/state/sites/features/actions.js

```diff
@@ -46,6 +50,44 @@ export function fetchSiteFeatures( siteId ) {
 	};
 }
 
+export function fetchJetpackSitesFeatures() {
+	return ( dispatch ) => {
+		dispatch( {
+			type: JETPACK_SITES_FEATURES_FETCH,
+		} );
+
+		return wpcom.req
+			.get( `/me/sites/features` )
```

Comment:
Nit: seems like there's no need for the string literal here:

```suggestion
			.get( '/me/sites/features' )
```